src/components/Accordion.tsx

```diff
@@ -0,0 +1,44 @@
+import { Css, IconButton, useTestIds } from "src/index";
+import { ReactNode, useState } from "react";
+
+export interface AccordionProps {
+  title: string;
```

Comment:
**suggestion:** Doing something like `title: string` has been where we started, but end up with `title: ReactNode`, because eventually someone wants to put in more than just text. Maybe another Icon, or Tag... So I would suggest allow this to be a `ReactNode` to start out with.